climada/hazard/centroids/centr.py

```diff
@@ -759,13 +763,18 @@ def select(self, reg_id=None, extent=None, sel_cen=None):
         cen : Centroids
             Sub-selection of this object
         """
+
         if sel_cen is None:
-            sel_cen = np.ones_like(self.region_id, dtype=bool)
+            sel_cen = np.ones_like(self.lat, dtype=bool)
             if reg_id:
                 sel_cen &= np.isin(self.region_id, reg_id)
             if extent:
-                sel_cen &= ((extent[0] < self.lon) & (extent[1] > self.lon)
-                            & (extent[2] < self.lat) & (extent[3] > self.lat))
+                lon_min, lon_max, lat_min, lat_max = extent
+                lon_max += 360 if lon_min > lon_max else 0
+                lon_normalized = lon_normalize(self.lon.copy(), center=0.5 * (lon_min + lon_max))
+                sel_cen &= ((lon_min <= lon_normalized) & (lon_normalized <= lon_max)
+                            & (lat_min <= self.lat) & (self.lat <= lat_max))
+
```

Comment:
I simplified your implementation.

Reply:
Awesome, now I start to understand the use of the `u_coord.lon_normalize`. I reorder slightly the code just to make for me more readable.

Reply:
Do you think we should follow the convention you/we started to use with the util functions? I.e. import them as `u_coord.lon_normalize` (or analog)? I could add that to this pull request for the centroids.

Reply:
This is addressed in #145 anyways, so you don't have to deal with it here.

Reply:
Amazing, alrady all the centr.py is done. Thanks! I will add it to the convention in the guide.